# 请问以下代码是否会产生异常,如果会的话,请写出异常的名称:
my_list = [3, 1, 5, 4, 2]
my_list.sorted()

p = open('C:\\test.txt', wb)
f.write('I love Python!\n')
f.close()


def my_fun1():
    x = 5
    def my_fun2():
        x *= x
        return x
    return my_fun2()
